src/components/Form/Input/CurrencyInput.tsx

```diff
@@ -19,7 +19,8 @@ function CurrencyInput(props: InputProps) {
   )
 
   useEffect(() => {
-    if (value) setLocalValue(getValueWithDecimalFormat(String(value)))
+    if (typeof value === 'string' || typeof value === 'number')
```

Comment:
It is necessary to be careful with this typeof === number, since it admits the NaN, it would be necessary to make some validation so that it does not admit it, what do you think @tbor00 ?

Reply:
how to recommend to validate it? The previous validation is modified because if the input was wanted to be set to null, it was not set and it remained with the previous value

Reply:
In that case, you can use !isNaN(value) to validate it

Reply:
In CurrencyInput and DateInput this validation is not necessary because they already go through a function that cleans the value, but in NumberInput and PercentageInput it would be necessary to add it

Reply:
I agree, we need validate this with some like that:

```js
!isNaN(value)
```